app/assets/stylesheets/navbar/components/_navbar_mobile.scss

```diff
@@ -0,0 +1,102 @@
+@mixin style1 {
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 5px;
+  width: 100px;
+}
+
+@mixin style2 {
+  border:2px solid white;
+  border-radius: 5px;
+}
+
+.navbar {
+  background-color: #08355e;
+
+  .navbar-brand {
+    width: 100px;
```

Comment:
What do you think about alphabetically accommodating the properties of your styles?